mysql/src/finagle_clojure/mysql/client.clj

```diff
@@ -190,7 +190,7 @@
   ([^PreparedStatement stmt params]
     (select-stmt stmt params Row->map))
   ([^PreparedStatement stmt params fn1]
-    (let [params (scala/seq->scala-buffer (map value/box params))
+    (let [params (scala/seq->scala-buffer (map value/parameterize params))
```

Comment:
would the existing `ToParameter` protocol work here? 
https://github.com/finagle/finagle-clojure/blob/cb941daa0e3dfda3f2c160179b4eda759af60508/mysql/src/finagle_clojure/mysql/client.clj#L226-L232